.github/workflows/build-test-distribute.yml

```diff
@@ -241,6 +245,61 @@ jobs:
          path: MeshLib.nupkg
          retention-days: 1
 
+     - name: Configure AWS Credentials
+       uses: aws-actions/configure-aws-credentials@v4
+       with:
+         role-to-assume: arn:aws:iam::259351611210:role/GitHubMeshLibAwsCredsRole
+         aws-region: us-east-1
+
+     - name: Upload NuGet Package to S3
+       shell: powershell
+       run: aws s3 cp MeshLib.nupkg 's3://test-artifacts-git/MeshLib.nupkg' --quiet
+
+  start-unity-instance:
+    name: Start unity instance
+    runs-on: aws-micro
+    needs: [ windows-build-test, create-nuget-package ]
+    outputs:
+      instance_started: ${{ steps.set-output.outputs.instance_started }}
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Configure AWS Credentials (OIDC)
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::259351611210:role/GitHubMeshLibAwsStartStopInstances
+          aws-region: us-east-1
+      - name: Start Instance
+        run: |
+          aws ec2 start-instances --instance-ids i-0987d7b35b21ad1f1
+      - name: Wait Instance
+        run: |
+          aws ec2 wait instance-status-ok --instance-ids i-0987d7b35b21ad1f1
```

Comment:
mb set `i-0987d7b35b21ad1f1` as constant?

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables